frontend/src/app/bitcoin.utils.ts

```diff
@@ -281,3 +281,12 @@ export function isFeatureActive(network: string, height: number, feature: 'rbf'
     return false;
   }
 }
+
+export async function calcScriptHash$(script: string): Promise<string> {
+  const buf = Uint8Array.from(script.match(/.{1,2}/g).map((byte) => parseInt(byte, 16)));
```

Comment:
Looping over match is fine, but it will produce some weird results if the input is weird.

```
> "aaf".match(/.{1,2}/g)
[ 'aa', 'f' ]

# even if we fix the regex
> "aaxxxxxxxxxxdd".match(/[0-9a-fA-F]{2}/g)
[ 'aa', 'dd' ]
```